### PR TITLE
Journal-page and journal issue page

### PR DESCRIPTION
**Pull request by ashwanisrivastav**
Journal-page and Journal issue page with global change

### CODE REVIEW

File: source/_patterns/03-components/site/_journal-article-as-issue.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// Journal Issue Page
+
+.journal-article.journal-page.journal-issue-page{
```
Comment (vinothk-hw) [inline source/_patterns/03-components/site/_journal-article-as-issue.scss:3]: not correct syntax. Please check sass and construct the same.. And one more thing "journal-page" keep as base and journal article page should -> "journal-page journal-article-page" and issue page like "journal-page journal-issue-page".

File: source/_patterns/03-components/site/_journal-article-as-page.scss [hunk 2]
```diff
@@ -0,0 +1,202 @@
+// Journal Article Page
+
+.journal-article.journal-page{
```
Comment (vinothk-hw) [inline source/_patterns/03-components/site/_journal-article-as-page.scss:3]: not sass format. should be 
.journal-article {
   &.journal-page{
    coding
   }
}

File: source/_patterns/03-components/site/_journal-article-as-page.scss [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+            content: "\e955";
+            font-family: "feather";
+            @extend .align-middle;
+            font-size: 1.5rem;
+            top: -1px;
+            color: $primary-dark;
+          }
+        }
+      }
+
```
Comment (vinothk-hw) [inline source/_patterns/03-components/site/_journal-article-as-page.scss:198]: unwanted spacing. this has been communicated already should not contain any extra space. Please check well before PR

**Review by vinothk-hw — COMMENTED** [review]
Please check update the same

File: source/_patterns/03-components/site/_journal-article-as-issue.scss (hunk 1 above)
Comment (ashwanisrivastav) [reply source/_patterns/03-components/site/_journal-article-as-issue.scss]: fixed

File: source/_patterns/03-components/site/_journal-article-as-page.scss (hunk 2 above)
Comment (ashwanisrivastav) [reply source/_patterns/03-components/site/_journal-article-as-page.scss]: done

File: source/_patterns/03-components/site/_journal-article-as-page.scss (hunk 3 above)
Comment (ashwanisrivastav) [reply source/_patterns/03-components/site/_journal-article-as-page.scss]: fixed

**Review by vinothk-hw — APPROVED** [review]

**Pull request merged**